R/file.R

```diff
@@ -92,9 +95,10 @@ file_copy <- function(path, new_path, overwrite = FALSE) {
 
 #' Change ownership or group of a file
 #' @template fs
-#' @param user_id The user id of the new owner, the R process must be privlaged to change
-#'   this.
-#' @param group_id The group id of the new owner
+#' @param user_id The user id of the new owner, specified as a numeric ID or
```

Comment:
Are numeric IDs only valid on windows?

Reply:
no ids are valid on windows, both uid and gid just return -1 on windows.